packages/viewer/src/lib/types/clippingPlanes.ts

```diff
@@ -1,29 +1,26 @@
 import { BoundingBox, Vector3 } from '@vertexvis/geometry';
 
+import { cameraToVector, distanceToVectorAlongViewVec } from '../scenes';
 import { FrameCamera } from './frameCamera';
 
 export interface ClippingPlanes {
   near: number;
   far: number;
 }
-
 export function fromBoundingBoxAndLookAtCamera(
   boundingBox: BoundingBox.BoundingBox,
   camera: FrameCamera
 ): ClippingPlanes {
   const boundingBoxCenter = BoundingBox.center(boundingBox);
-  const cameraToCenter = Vector3.subtract(camera.position, boundingBoxCenter);
+  const cameraToCenter = cameraToVector(boundingBoxCenter, camera);
   const centerToBoundingPlane = Vector3.subtract(
     boundingBox.max,
     boundingBoxCenter
   );
-  const distanceToCenterAlongViewVec =
-    Math.abs(
-      Vector3.dot(
-        Vector3.subtract(camera.lookAt, camera.position),
-        cameraToCenter
-      )
-    ) / Vector3.magnitude(Vector3.subtract(camera.lookAt, camera.position));
+  const distanceToCenterAlongViewVec = distanceToVectorAlongViewVec(
+    cameraToCenter,
+    camera
+  );
   const radius = 1.1 * Vector3.magnitude(centerToBoundingPlane);
   let far = distanceToCenterAlongViewVec + radius;
   let near = far * 0.01;
```

Comment:
What are these used for?  I'm a little concerned that this logic does not match up with RCL's version....

Reply:
Yeah, I don't know that this logic has been revisited any time recently. I think the only logic around the `near` and `far` clipping planes is depth buffer related. We do have https://vertexvis.atlassian.net/browse/PLAT-1151 which appears to be a regression in the "rotate-point" interaction to rotate around a point computed from the depth buffer, so I wonder if that's related? I can take a look at updating the logic with this PR at least.

This logic was added to the client thinking along the same lines as the camera matrices - essentially that we can compute the `near` and `far` plane values on the client, so omit it from the payload to reduce size. Given the amount of reliance we have on those values now with depth buffers and the potential for them to change though, maybe worth revisiting + creating a follow-up ticket to remove this calculation

Reply:
If this is being used to map values into the depth buffer, then the logic absolutely _must_ match.  

Overall, we need to decide how these values get calculated -- ideally, it is done in exactly one place to avoid mismatches but maybe that's not practical.

Reply:
If this is being used to map values into the depth buffer, then the logic absolutely _must_ match.  

Overall, we need to decide how these values get calculated -- ideally, it is done in exactly one place to avoid mismatches but maybe that's not practical.

Reply:
Created https://vertexvis.atlassian.net/browse/PLAT-1187 and put it in grooming to discuss this, but also updated the near/far computation with https://github.com/Vertexvis/vertex-web-sdk/pull/349/commits/dcc664815a0376579629810fa88a7e8cfc6fc026